amethyst_core/src/transform/systems.rs

```diff
@@ -60,18 +60,23 @@ impl<'a> System<'a> for TransformSystem {
         self.global_modified.clear();
 
         locals.populate_inserted(
-            self.inserted_local_id.as_mut().unwrap(),
+            self.inserted_local_id
+                .as_mut()
+                .expect("'TransformSystem::setup' was not called before 'TransformSystem::run'"),
```

Comment:
This should use a backtick (\`) instead of a single quote (').

Reply:
In an error message?

Reply:
Why do you feel backticks are not appropriate in error messages? It's just the standard style we use for code there.

Reply:
Hmm good question. Change is on the way